docs/server.rst

```diff
@@ -36,12 +36,54 @@ Prerequisites
 -------------
 
 The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
-tunnels. The following modules are required for Tunneldigger operation:
+tunnels.
+
+These distributions are supported:
+
+* Debian
+* Fedora with the package ``kernel-modules-extra``
+* *add your distribution*
+
+Kernel Modules
+~~~~~~~~~~~~~~
+
+The following modules are required for Tunneldigger operation:
 
 * ``l2tp_core``
 * ``l2tp_eth``
 * ``l2tp_netlink``
 
+You can find out if these modules are loaded by running ``lsmod | grep l2tp``.
+If you get no output, they need to be activated.
+For the activation of the kernel modules, we recommend adding a file
+``/etc/modules-load.d/tunneldigger.conf`` with the following content:
+
+.. code:: shell
+
+    l2tp_core
+    l2tp_eth
+    l2tp_netlink
+
+You can also activate the modules using modprobe.
+A restart will load the kernel without them if you do not create the file.
+
+.. code:: shell
+
+    $ sudo modprobe l2tp_core
+    $ sudo modprobe l2tp_eth
+    $ sudo modprobe l2tp_netlink
+
+Then, your listing of the modules can look like this:
+
+.. code:: shell
+
+    $ lsmod | grep l2tp
+    l2tp_eth               16384  0 
+    l2tp_netlink           24576  1 l2tp_eth
+    l2tp_core              32768  2 l2tp_eth,l2tp_netlink
+    ip6_udp_tunnel         16384  1 l2tp_core
+    udp_tunnel             16384  1 l2tp_core
+
 In addition the kernel must support network address translation via netfilter,
 otherwise the tunnels will not work as Tunneldigger uses translation to achieve
 that all tunnels operate over the same external port.
```

Comment:
There is no distro wirth mentioning that does not support netfilter, right?

I think I'd like to see this and the next 2 paragraphs removed. They are just confusing people by pointing out a problem that is extremely unlikely to exist.

Reply:
1d208e5 covers this